data/json/monsters/zed_misc.json

```diff
@@ -938,7 +938,7 @@
   {
     "id": "mon_zombie_fursuit",
     "type": "MONSTER",
-    "name": { "str": "malicious mane" },
+    "name": { "str": "fursuited zombie" },
     "//": "A pun. 'Mane' being an archaic synonym of zombie, which also refers hair on the back of an animals neck.",
```

Comment:
It is a pun.

Reply:
It is also a pretty forceful alliteration, and a name that doesn't give much of a clue to what it actually is. Also, I've never heard "mane" being used for zombies, and I doubt much people have either

Reply:
Honestly this one should stay, its way funnier with the strange name.  Honestly its better removed if the name MUST change,

Reply:
Well, it is in a dictionary.

Reply:
also it's quite funny removing the pun but not the comment that explains the pun.

Reply:
> Honestly this one should stay, its way funnier with the strange name. Honestly its better removed if the name MUST change,

I think it's better removed, mainly because even from blob psychosis, I don't think people would waste a thousands of dollars suit /j
But it's inclusion does feel quite odd to me to be honest. I don't think during riots people would attend a furry convention, or don a fursuit for any other reason